bindings/CXX11/cxx11/IO.h

```diff
@@ -240,6 +240,15 @@ class IO
     /** Flushes all engines created with this IO with the Open function */
     void FlushAll();
 
+    /**
+     * @brief Promise that no more definitions or changes to defined variables
+     * will occur. Useful information if called before the first EndStep() of an
+     * output Engine, as it will know that the definitions are complete and
+     * constant for the entire lifetime of the output and may optimize metadata
+     * handling.
+     */
+    void LockDefinitions();
```

Comment:
missing noexcept: `void IO::LockDefinitions() noexcept`

Reply:
Thanks, I'll fix it.